niceplots/utils.py

```diff
@@ -61,7 +61,13 @@ def get_colors():
     -------
     dict
         Dictionary of the colors for the requested style. The keys
-        are human-readable names and the keys are the hex codes.
+        are human-readable names and the keys are the hex codes. It
+        also adds color names from the rcParams that are generally useful:
+
+            - "Axis": axis spine color
```

Comment:
Not a big deal but I would prefer if these are not capitalized

Reply:
I think it makes sense to keep capitalized for consistency because the color names are also capitalized

Reply:
Yeah I would argue that the colours shouldn't be capitalized either (as done in matplotlib) but alright.